testing/integration/Cargo.toml

```diff
@@ -8,7 +8,7 @@ authors = ["Protocol Labs", "Filecoin Core Devs", "Polyphene"]
 repository = "https://github.com/filecoin-project/ref-fvm"
 
 [dependencies]
-fvm = { version = "3.9.0", path = "../../fvm", default-features = false, features = ["testing"] }
+fvm = { version = "3.10.0", path = "../../fvm", default-features = false, features = ["testing"] }
 fvm_shared = { version = "3.6.0", path = "../../shared", features = ["testing"] }
```

Comment:
need to update fvm_shared. Unlike in v4, in v3 we're not (yet) using the workspace itself to track versions (patches welcome).

Reply:
Also need to release shared.

Reply:
I updated fvm_shared to 3.1.0.0 in https://github.com/filecoin-project/ref-fvm/pull/2016/commits/1024200283a93f028e949cbe15fd4d750a5a1ca3, but it seems like that caused CI to fail. So I thought I had to run `cargo check --all` but that failed with:

```
error: failed to select a version for the requirement `fvm_shared = "^3.10.0"`
candidate versions found which didn't match: 3.6.0
location searched: /Users/phi/ref-fvm/shared
required by package `fvm_integration_tests v3.3.0 (/Users/phi/ref-fvm/testing/integration)`
```

Any ideas for how I can get this to pass after updating fvm_shared? Sorry for asking so many novice questions, not that familiar with ref-fvm.

Reply:
You need to update the `fvm_shared` version. Unlike in master (v4), we haven't moved the versions to the root cargo.toml.

Reply:
(so `fvm` and `fvm_shared` can actually have different versions)

Reply:
(if you have some time and want to backport the workspace versioning changes from master, I'd appreciate it)